phpunit.xml.dist

```diff
@@ -17,4 +17,8 @@
             <directory>src</directory>
         </whitelist>
     </filter>
+
+    <logging>
+        <log type="coverage-clover" target="build/coverage/clover.xml"/>
```

Comment:
Right, forgot about the command line switch. I want to use it on travis (in combination with scrutinizer), if everyone else can agree to that as well.